.github/workflows/ci.yml

```diff
@@ -542,6 +542,65 @@ jobs:
             build/*.cmake
             build/testsuite/*/*.*
 
+  icx-vp2022:
+    name: "Linux VFXP-2022 icx/C++17 llvm13 py3.9 boost1.76 exr3.1 oiio2.3 avx2"
+    runs-on: ubuntu-latest
+    container:
+      image: aswftesting/ci-osl:2022-clang13
+    env:
+      CXX: icpx
+      CC: icx
+      CMAKE_CXX_STANDARD: 17
+      FMT_VERSION: 7.1.3
+      OPENIMAGEIO_VERSION: master
+      OPENIMAGEIO_CMAKE_FLAGS: -DBUILD_FMT_VERSION=7.1.3
+      PYBIND11_VERSION: v2.9.0
+      PYTHON_VERSION: 3.9
+      USE_BATCHED: b8_AVX2_noFMA
+      # USE_BATCHED: b16_AVX512
+      # USE_SIMD: avx2,f16c
```

Comment:
I had commented out the `USE_SIMD` for icc because it uses a different syntax than the `-m` arguments that this gets turned into by compiler.cmake. But icx accepts the same `-m` arguments as clang. So you can re-enable that line for icx.

Reply:
I'll try adding USE_SIMD: avx2,f16c back in

Reply:
It worked for me on the OIIO side.

I've been contemplating a change to the logic in compiler.cmake to basically be sure to do the right thing to the USE_SIMD for icc under the covers (i.e., if it sees USE_SIMD=avx512f, instead of just passing on `-mavx512f`, which I think is not recognized by icc, it would translates to `-axCORE-AVX512` [I think]). But now that I see that icx recognizes all the same commands as clang, I'm wondering if extra work for icc is really worth the trouble (or if users should just use EXTRA_CPP_ARGS to set the right thing if they know it) if icc users are going to tend to switch to icx over time anyway.

refs:
https://www.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top/compiler-reference/compiler-options/compiler-option-details/code-generation-options/m.html
https://www.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top/compiler-reference/compiler-options/compiler-option-details/code-generation-options/x-qx.html

Reply:
Not a make file guru, if passing native flags, then just make user pass them in the 1st place.  Or have those be OSL defined and controlled flags that get translated.  Technically adding "-m" does count as translating, so having it spit out "-axCORE-AVX512" seems totally reasonable.  You do know the "-ax" is automatic dispatch that generates multiple versions of functions with a spring board to jump to the correct one based on CPUID?